### DISCUSSION TITLE
Bi-Weekly Builders Call - Agendas and Topic Proposals

### DISCUSSION THREAD

**Discussion by lufa23** [archway-network/discussions#12] [General Discussions]
This Discussion is to provide major visibility to the Agendas of the Weekly Builders Calls, as well as providing a way for developers to propose topics to be discussed :wrench: :sunglasses:

Weekly Builders Calls take place every Wednesday at 15:00 UTC on Discord, in the [builders-calls voice channel](https://discord.gg/brmPdBcw).
Make sure you get a _Developer_ role in the [get-a-role channel](https://discord.gg/archwayhq)

**Comment by lufa23** [comment]
# Weekly Builders Call #8

**Date:** 13/09/2023

**Time:** 15:00 UTC

**Duration:** 60 mins

**Where:** Discord, **[builders-call](https://discord.gg/brmPdBcw)** voice channel

## NetX Discussion

- How to use the relayer_exporter to monitor IBC clients expirations
- Other work in progress in open-sourced repositories
- Feedback and Q&As

## Dapp Showcase: [Data Pool (Kyve Network)](https://app.kyve.network/#/pools/2)

- Showcase of Data Pools codebase
- Feedback and final Q&As

**Comment by lufa23** [comment]
# Weekly Builders Call 9
**Date**: 20/09/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call](https://discord.gg/brmPdBcw) voice channel

## Protocol Discussion
- [Allow Fee Grants for Whitelisted Contracts](https://github.com/CosmWasm/wasmd/issues/1571)
- [Add a x/cw-registry module](https://github.com/orgs/archway-network/discussions/16)
- Feedback and Q&As

## Dapp Showcase: [Nomos Multisig](https://github.com/CosmWasm/wasmd/issues/1571)
Guest: [Chelo Labs](https://chelo.fi/)

- Showcase of Nomos codebase
- Feedback and final Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 10
**Date**: 27/09/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call](https://discord.com/events/892203409418092615/1156070119894691881)

## Dapp Showcase: Ojo
Guest: [Ojo Network](https://ojo.network/)

- Showcase of Ojo's Oracle network
- Feedback and Q&As


## Protocol Discussion
- MultiSig solution
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 11
**Date**: 04/10/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, builders-call voice channel

## Dapp Showcase: [SudoStake](https://sudostake.vercel.app) (30 minutes)
Guest: Palingram

- Showcase SudoStake - Non-Custodial, Smart Contract Staking, Peer-to-Peer, Options Trading Platform
- Feedback and Q&As


## DevX Discussion
Guest: Adrian Thompson

- Archway CLI v2 demo of the Workspaces feature
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 12
**Date**: 11/10/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1160983449818103830)

## Dapp Showcase: [evolvNFT](https://evolvnft.com) (30 minutes)
Guest: Michael

- Showcase of the evolvNFT platform - a decentralized hub that allows you to easily create, manage and evolve your decentralized assets
- Feedback and Q&As


## PhiLabs Team Presentation: Protocol (30 minutes)
Guest: Spoorthi Satheesha

- Discussion regarding an RFC for a fresh x/cwcallback module (https://github.com/orgs/archway-network/discussions/25).
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 13
**Date**: 18/10/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.gg/archwayhq?event=1163866080712867851)

## PhiLabs Team Presentation: Protocol (20 minutes)
Guest: Billa

- Discussion around the contract registry (https://github.com/archway-network/contract-register).
- Feedback and Q&As

## Dapp Showcase: [NodeOps](https://nodeops.xyz) (30 minutes)
Guest: Naman

- Showcase of the NodeOps platform - NodeOps provides AI enabled infrastructure specifically curated for Node Operators and Developers and designed to simplify the development process and operations for Web3 protocols.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 14
**Date**: 25/10/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1166373892953219072)

## PhiLabs Team Presentation: Devx (20 minutes)
Guest: Augusto

- Discussion around the new Archway Bindings and Protobuf packages (https://github.com/archway-network/arch3.rs/tree/main/packages).
- Feedback and Q&As

## Dapp Showcase: [Abstract](https://abstract.money) (30 minutes)
Guest: Adair

- Showcase of the Abstract platform and tooling - The Abstract platform offers a suite of CosmWasm-focused products, including an on-chain smart-contract infrastructure (Abstract SDK), development tooling (cw-orchestrator), and front-end libraries (Abstract.js). These products are designed for composability, allowing developers to easily reuse the components necessary to create their applications. Abstract aims to streamline the development process while providing a powerful tool to foster innovation with minimal effort.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 15
**Date**: 1/11/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1167169271323955230)

## Interactive Workshop: [Abstract](https://abstract.money) (60 minutes)
Guest: Adair

The workshop will be centered around their **cw-orchestrator** tool (https://github.com/AbstractSDK/cw-orchestrator) which is an advanced testing and deployment tool for CosmWasm smart-contracts.

**Comment by emperorjm** [comment]
# Weekly Builders Call - 16
**Date**: 8/11/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1169367104957460651)

## PhiLabs Team Presentation: Protocol (60 minutes)
Guest: Spoorthi

This will be an in-depth session to discuss the implications of the new 5.x upgrades and to ensure that your projects continue to run smoothly on the updated protocol. The following discussion provides some insight https://github.com/orgs/archway-network/discussions/27

**Comment by emperorjm** [comment]
# Weekly Builders Call - 17
**Date**: 15/11/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1171974226178224259)

## Dapp Showcase: [Andromeda](https://www.andromedaprotocol.io/) (60 minutes)
Guest: Mark

- Showcase of the Andromeda platform - Andromeda is an all-on-chain suite of products, tools, and utilities enabled by a decentralized operating system we call aOS, or Andromeda Operating System. aOS is designed to make Web3 Easier, Better, Faster. The following video gives an overview of the Andromeda platform https://www.youtube.com/watch?v=Jw6h7D9FhwE.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 18
**Date**: 22/11/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.gg/archwayhq?event=1176192544376373288)

## Dapp Showcase: [Chelo](https://chelo.fi) (60 minutes)
Guest: Ohad

- Showcase of the new Nomos platform - A multisig wallet solution.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 19
**Date**: 29/11/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1177308334773244014)

## Platform Showcase: [Subquery](https://subquery.network/) (60 minutes)
Guest: Boris

- Showcase of the Subquery platform - SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. See the docs for integration with Archway: https://docs.archway.io/developers/developer-tools/subquery
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 20
**Date**: 06/12/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1180238841609723994)

## Platform Showcase: [Leap Wallet](https://www.leapwallet.io/) (30 minutes)
Guest: Mayank, Engineering Lead at Leap Wallet

- Leap Wallet (extension and mobile) on a dapp Metamask snaps 
- Elements and embedded Wallet
- Feedback and Q&As

## Platform Showcase: [OmniFlix](https://omniflix.network/) (30 minutes)
Guest: Sistla Abhishek, Core Contributor

- Showcase of the OmniFlix platform
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 21
**Date**: 13/12/2023

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1184136576473776241)

## Platform Showcase: [ArchMon](https://www.archmonnft.xyz/) (30 minutes)
Guest: Ali Zahreddine [Co-Founder of ArchmonNFT]

- Overview of Archmon: Brief introduction to our team and the concept behind ArchmonNFT.
- Dynamic NFT Evolution: Highlighting how user interactions with dApps lead to NFT evolution, enhancing user engagement.
- Experience System: Explaining our XP and mission-based system that allows NFTs to unlock new features.
- Community Integration and Future Vision: Discussing our goals for community involvement and our long-term vision for Archmon as a central hub in blockchain gaming.
- Feedback and Q&As

## Platform Showcase: [KYVE](https://www.kyve.network/) (30 minutes)
Guest: Troy Kessler, Lead Protocol Developer at KYVE

- Introducing KYVE and its trustless backup of the entire Archway chain. Along with a demo of its latest data syncing tool KSYNC, enabling  syncing to any Archway block, state snapshot, or height in just a few minutes for efficient and reliable building and network participation
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 22
**Date**: 10/01/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1194026958145732718/1194656897433600000)

## Platform Showcase: [OmniFlix](https://omniflix.network/) (30 minutes)
Guest: Sistla Abhishek, Core Contributor

- Showcase of the OmniFlix platform
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 23
**Date**: 17/01/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1194670929796661259/1197193612492800000)

## Platform Showcase: [Reclaim](https://www.reclaimprotocol.org) (60 minutes)
Guest: Ruchil Sharma

- Showcase of what Reclaim has to offer
- Feedback and Q&As

**Reply by emperorjm** [reply]
Additional resources:

- Official Website: https://www.reclaimprotocol.org
- Read the Whitepaper: https://drive.google.com/file/d/1wmfdtIGPaN9uJBI1DHqN903tP9c_aTG2/view
- Ideas for Building: https://blog.reclaimprotocol.org/posts/usecases
- Watch Demo 1: https://www.loom.com/share/c54d572a1e894395b5160cfad6bb3e08
- Watch Demo 2: https://www.loom.com/share/b88ab8bb90ff498c8cbd486bdb4a42a4

**Comment by emperorjm** [comment]
# Weekly Builders Call - 24
**Date**: 24/01/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1197210490502402128/1199730327552000000)

## Platform Showcase: [Gelotto](https://demo.gelotto.io/app/community/threads) (60 minutes)

- Showcase of the new Threads platform. The project is currently in its ALPHA stages. Threads is a social media platform similar to Reddit and is accessible at https://demo.gelotto.io/app/community/threads. It is now live on Mainnet, and you are welcome to explore it.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 25
**Date**: 31/01/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1194670929796661259/1197193612492800000)

## Protocol Discussion (60 minutes)
Guest: Billa Coetsee

- Will give an overview of the upcoming upgrades for this quarter which includes Withdrawal Enhancements, SDK 0.47 upgrade, Callback Module and CW FeeGrant.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 26
**Date**: 21/02/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1209157208135307264/1209877187788800000)

## Platform Showcase: [ChainIDE](https://chainide.com) (60 minutes)

- ChainIDE is a Cloud-Based Multi-Chain IDE offering a suite of services for dapp development, including debugging, testing, and deployment, without the need for additional installations.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Weekly Builders Call - 27
**Date**: 28/02/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1209157208135307264/1212413902848000000)

## Platform Showcase: [Kado](https://www.kado.money) (60 minutes)

Presenter: Max Batty

- Showcase of the Kado platform and integrations for dapps
  - Kado makes it easy to convert fiat to crypto using various payment methods. Bypass centralized exchanges and transfer funds directly to your self-custody wallet.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Bi-Weekly Builders Call - 28
**Date**: 12/03/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1212432701970190436/1217487332966400000)

## Platform Showcase: [ChainRisk](https://www.chainrisk.cloud) (60 minutes)

Presenter: Arka Datta

- Showcase of the ChainRisk platform. ChainRisk is the world's first Economic Risk Management SaaS for DeFi protocols.
- Feedback and Q&As

**Comment by emperorjm** [comment]
# Bi-Weekly Builders Call - 29
**Date**: 27/03/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1218893762080669778/1222560763084800000)

## Platform Showcase: [Nomos](https://nomos.ms) (60 minutes)

Presenters: Ohad, Chanaka, and Rafa

Agenda:

1. Demonstration of ICA creation from Archway to Osmosis
2. Discussion Topics:
   - How ICA works
   - Unique aspects of Nomos development
   - Various use cases for ICA and interchain interoperability
   - Risks associated with ICA and potential solutions

**Comment by emperorjm** [comment]
# Bi-Weekly Builders Call - 30
**Date**: 10/04/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1222582568294285382/1227634193203200000)

## Platform Showcase: [Ekza Space](https://ekza.io) (60 minutes)

Presenters: **Wotori**

Description: **Ekza** is a collaborative platform and a metaverse playground that redefines content creation. It invites users to join forces in crafting innovative content and leverages their creations across Ekza's ecosystem and its partners' metaverses and games. At its core, Ekza empowers its community by offering continuous earnings from their contributions, ensuring creators receive a distributed share of the revenue generated as their work is utilized by others.

Agenda:
  - Collaboration interface
  - Metaverse engine
  - Smart contracts behind the interface
  - Pitchdeck - key slides/ideas

**Comment by emperorjm** [comment]
# Bi-Weekly Builders Call - 31
**Date**: 24/04/2024

**Time**: 15:00 UTC

**Duration**: 60 mins

Where: Discord, [builders-call voice channel](https://discord.com/events/892203409418092615/1227672683916038145/1232707623321600000)


## PhiLabs Team: Protocol (30 minutes)
Guest: Spoorthi Satheesha

**Agenda**:

- Discuss the upcoming Version 7 protocol upgrade which will introduce four new modules: [Callback](https://github.com/archway-network/archway/blob/main/docs/adr/ADR-009-callback-module.md), [Fees](https://github.com/archway-network/archway/blob/main/docs/adr/ADR-010-cw-fees.md), [ICA](https://github.com/archway-network/archway/blob/main/docs/adr/ADR-011-cw-ica.md) and [Errors](https://github.com/archway-network/archway/blob/main/docs/adr/ADR-012-cw-errors.md)
- Feedback and Q&As
